src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private static final FigureName figureName = FigureName.SQUARE;
```

Comment:
The `figureName` constant should be a part of the `FigureName` enum, not a static variable in each figure class. Remove it from here and use `FigureName.SQUARE.name()` when you need a string representation of the figure name.